recipes/i2c-tools/all/test_package/example.cpp

```diff
@@ -0,0 +1,4 @@
+#include "i2c/smbus.h"
+
+int main() {
```

Comment:
Please exercise the library: use some symbols.

Reply:
I'm not (at all) very familiar with the library, as I said in the PR I only need it in order to get another library.
I found some examples online, but do you perhaps think the code you provided in your PR to my branch is sufficient?
Oh and thanks a bunch by the way! :)

Reply:
A test package is just a proof that the library can be used.
I copied the way of initialization from a source inside the tar ball.

No problem.

Reply:
I guess it's fine then? All I have to do now is to wait for my user to get access to CI to run and build?

Reply:
Yup. They do that in batches every week.
So if in 1 week you've not been accepted yet, post something here.

Reply:
Great, then I will wait :)